### PR TITLE
[ja] add tasks/configure-pod-container/security-context.md

### LINKED ISSUE #30519: [ja] Translate tasks/configure-pod-container/security-context into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/configure-pod-container/security-context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by FeLvi-zzz**
Add desrtiption of Security Context for Pod.
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

close #30519

### CODE REVIEW

File: content/ja/docs/tasks/configure-pod-container/security-context.md
Comment (s-kawamura-w664) [inline content/ja/docs/tasks/configure-pod-container/security-context.md:30]:
```suggestion
  コンテナが以下の場合、`allowPrivilegeEscalation`は常にtrueになります。
  - コンテナが特権で動いている、または
  - `CAP_SYS_ADMIN`を持っている
```
I think it would be better to adapt the format to en-site.

File: content/ja/docs/tasks/configure-pod-container/security-context.md
Comment (s-kawamura-w664) [inline content/ja/docs/tasks/configure-pod-container/security-context.md:52]:
```suggestion
`runAsGroup`フィールドはPod内のコンテナに対して全てのプロセスをプライマリーグループID 3000で実行するように指定します。このフィールドが省略されたときは、コンテナのプライマリーグループIDはroot(0)になります。`runAsGroup`が指定されている場合、作成されたファイルもユーザー1000とグループ3000の所有物になります。
```
I understood as above. Is it correct?

**Review by s-kawamura-w664 — COMMENTED** [review]
I have not confirm all yet, but I leave some comments.

File: content/ja/docs/tasks/configure-pod-container/security-context.md
Comment (s-kawamura-w664) [inline content/ja/docs/tasks/configure-pod-container/security-context.md:385]: The literal translation is "リラベリング" or " 再ラベルづけ", but I understood that the re does not need to be translated to make sense. Am I correct in my understanding?

**Review by s-kawamura-w664 — COMMENTED** [review]
I have finished checking all of this file.
Please find my comments.

File: content/ja/docs/tasks/configure-pod-container/security-context.md
Comment (s-kawamura-w664) [inline content/ja/docs/tasks/configure-pod-container/security-context.md:442]:
```suggestion
* [AllowPrivilegeEscalation design document](https://git.k8s.io/design-proposals-archive/auth/no-new-privs.md)
```
There is line break In the English document , but I think that it would be better to have no line break.

**Review by s-kawamura-w664 — COMMENTED** [review]
One comment was left out and will be added.

File: content/ja/docs/tasks/configure-pod-container/security-context.md
Comment (FeLvi-zzz) [reply content/ja/docs/tasks/configure-pod-container/security-context.md:52]: yes, I misread. Thanks!

File: content/ja/docs/tasks/configure-pod-container/security-context.md
Comment (FeLvi-zzz) [reply content/ja/docs/tasks/configure-pod-container/security-context.md:385]: I thought "ラベリング" was easier to understand. However, there are many examples of "再ラベル付け" as a result of Google search. It seems that "再ラベル付け" is used widely, so I'll rewrite it with "再ラベル付け".

Comment (s-kawamura-w664) [reply content/ja/docs/tasks/configure-pod-container/security-context.md]: Thank you for your changing.
I think that your choise is good.

**Comment by s-kawamura-w664** [comment]
Thank you for your update.
The updates are fine.
/lgtm

**Review by t-inu — APPROVED** [review]
LGTM

**Comment by atoato88** [comment]
/assign @nasa9084 
Could you review this and approve if ok?

File: content/ja/docs/tasks/configure-pod-container/security-context.md
Comment (nasa9084) [inline content/ja/docs/tasks/configure-pod-container/security-context.md:183]:
```suggestion
コンテナに対してセキュリティ設定を行うには、コンテナマニフェストに`securityContext`フィールドを含めてください。`securityContext`フィールドには[SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)オブジェクトが入ります。
```

**Comment by FeLvi-zzz** [comment]
@nasa9084 
It hasn't merge yet, does it need a lgtm label to merge this PR? If so, could you attach one?

**Comment by nasa9084** [comment]
yes, need a lgtm label and it should be attached by reviewers again. please wait being reviewed again

**Comment by s-kawamura-w664** [comment]
Thanks for the update, I think it's good.
/lgtm

**Pull request merged**
